kodi_addon_checker/check_artwork.py

```diff
@@ -26,9 +27,53 @@ def check_artwork(report: Report, addon_path: str, parsed_xml, file_index: list,
         :parsed_xml: xml file i.e addon.xml
         :file_index: list having name and path of all the files in an addon
     """
-    art_type = ['icon', 'fanart', 'screenshot']
-    for image_type in art_type:
-        _check_image_type(report, image_type, parsed_xml, addon_path, kodi_version)
+    Asset = namedtuple('Asset', ['image_type', 'specifications'])
+
+    art_assets = [
+        Asset(
+            'icon',
+            {
+                'extension': ['.png'],
+                'transparency': False,
+                'sizes': [(256, 256), (512, 512)]
+            }
+        ),
+        Asset(
+            'fanart',
+            {
+                'extension': [".jpg", ".jpeg"],
```

Comment:
File size seems to be the reason to limit this, would it be possible to include png and set a max file size(150-200kb?)?
I'm using a 3840x2160 .png for the Twitch add-on which is only 32kb
https://github.com/MrSprigster/Twitch-on-Kodi/blob/master/fanart.png

Reply:
Probably not a bad idea to not limit so much on formats and use the root cause of the specification: filesize. I'll see what I can do.

Reply:
Thinking about this a bit more the filesize check should be an easy one. Allowing fanarts in png should only be done for krypton and above versions, gotham-krypton have the fanart.jpg and icon.png fallback in the root of the addon, so we need to somehow relax the check while not overcomplicating the code (the linter is pretty anoying on this project :P)